src/ModuleA.lua

```diff
@@ -363,14 +363,14 @@ function M.build_availA(self)
    local function build_availA_helper(mpath, sn, v, A)
       local icnt = #A
       if (v.file ) then
-         if (show_hidden or mrc:isVisible(sn)) then
+         if (show_hidden or mrc:isVisible({fullName=sn,sn=sn,fn=v.file})) then
```

Comment:
On this one, I'm not sure. What is the `fullName` and `sn` here? They seem to be equal?